pypln/web/settings.py

```diff
@@ -205,5 +194,24 @@
 
 
 ELASTICSEARCH_CONFIG = {
-    'hosts': ['127.0.0.1'],
+    'hosts': config('ELASTICSEARCH_HOSTS', cast=Csv()),
```

Comment:
`ELASTICSEARCH_HOSTS` probably needs a default value (`127.0.0.1` should be fine).